packages/client/.eslintrc.js

```diff
@@ -1,9 +1,9 @@
 module.exports = {
   plugins: ['@typescript-eslint', 'prettier', 'emotion', 'react', 'react-hooks'],
   extends: ['plugin:react/recommended', '../../.eslintrc.js', 'plugin:prettier/recommended'],
-  plugins: ['@typescript-eslint', 'prettier'],
+  // plugins: ['@typescript-eslint', 'prettier'], // revert a4963401
   parserOptions: {
```

Comment:
look like https://github.com/ParabolInc/parabol/commit/a4963401 broke the client lint command. i get lots of `Definition for rule was not found` errors. reverting to the earlier plugins fixes the issue, but it means we're no longer supporting eslint v7+.

Reply:
if this change is accepted, we should create a separate issue for supporting eslint v7+ in client package?